barretenberg/cpp/src/barretenberg/barretenberg.hpp

```diff
@@ -15,6 +15,7 @@
 #include "crypto/keccak/keccak.hpp"
 #include "crypto/pedersen_commitment/pedersen.hpp"
 #include "crypto/pedersen_hash/pedersen.hpp"
+#include "crypto/poseidon2/poseidon2.hpp"
```

Comment:
added in an attempt to solve linker error, but probably still needed

Reply:
seems like this file is unused so adding this is just for documentation about what is in barretenberg